archive/demos/shiny-drum-machine-data.js

```diff
@@ -0,0 +1,328 @@
+function freeze(obj) {
```

Comment:
nit: This uses 4 spaces indentation. I hope we could follow Google's JS style guide. (2 spaces) Better yet, we could easily apply the eslint-config-google. What do you think?

Reply:
Done.